.github/workflows/build.yml

```diff
@@ -20,10 +20,11 @@ jobs:
       - name: Check release version
         id: check-tag
         run: |
-          echo ::set-output name=version::$(echo ${{ github.event.ref }} | cut -d / -f 3 | cut -c2-)
+          echo version=$(echo ${{ github.event.ref }} | cut -d / -f 3 | cut -c2-) >> $GITHUB_OUTPUT
           if [[ ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-              echo ::set-output name=match::true
+              echo match=true >> $GITHUB_OUTPUT
           fi
+        shell: bash
```

Comment:
If you want to run your platform on Windows with CI in the future
you will need to set up a shell.

Reply:
Thanks @k1rnt for upgrading this code after GitHub's recent announcement.

Would you mind adding double quotes around `{environment_variable_name}={value}` (here and in `deploy.yaml` too) so the new code matches [GitHub's documentation](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files)?

Then I am not sure about the Windows CI part. Let's not optimize prematurely. The CI works fine in its current state. I don't think we need the `shell` lines, at least in this PR. If you think it is a good practice, let's open a different issue and a different PR to discuss the matter. I like to keep concerns separated as much as possible.

Reply:
@lvaylet thanks for the request for change and suggestion for correction!

[fix: double quotes around](https://github.com/google/slo-generator/pull/252/commits/b1aea834b8bb7358c5e7ba1f15f6b8ec4cef209b)
[revert set shell](https://github.com/google/slo-generator/pull/252/commits/198706e632bd20a2fa0cfa98a52220e6bc7618ca)

I have corrected the points you pointed out in this commit.

The Windows CI was also not currently needed as you commented.
If you ever need it in the future, I'd be happy to help!